Source/ChocolateyGui/ViewModels/SettingsViewModel.cs

```diff
@@ -270,6 +272,26 @@ public void Back()
 
             var sources = await _packageService.GetSources();
             Sources.AddRange(sources);
+        }
+
+        private void LocalizateFeaturesDescription(ChocolateyFeature feture)
+        {
+            var key = string.Format("SettingsViewModel_FeatureDescription_{0}", feture.Name);
+            var description = Resources.ResourceManager.GetString(key, Resources.Culture);
+            if (!string.IsNullOrWhiteSpace(description))
+            {
+                feture.Description = description;

```

Comment:
treating warnings as errors (at least in 'Release' configuration) sounds like a good idea!
- shouldn't be to complicated to clean up the current warnings?